chapters/en/module1.md

```diff
@@ -12,367 +12,149 @@ id: 1
 
 </opt>
 
 </choice>
 
 
 **Question 2**      
-What kind of components are combined in declarative plotting to create visualizations?
+What can you do with Jupyter Book?
 
 <choice id="2" >
-<opt text="Dictionary Keys">
+<opt text="Version control files">
 
-This is a data structure and not quite a plotting structure.
+Git is the tool that allows you to version control files.
 
 </opt>
 
-<opt text="Block Components">
+<opt text="Navigate the file system">
 
-Think more language-oriented.
+For this you need a shell, such as Bash.
 
 </opt>
 
-<opt text="Grammatical Components"  correct="true">
+<opt text="Publish books and web pages"  correct="true">
 
 You got it!
 
 </opt>
 
+<opt text="Edit code files">
 
-<opt text="Grouping Components">
-
-I'm not entirely sure what a grouping component is since we made this up!
+We need a text editor or programming environment such as JupyterLab or VSCode for this!
 
 </opt>
 
 </choice>
 
+**True or False**       
+*All the tools we are using in this course are open source and free to use.*
 
-</exercise>
-
-<exercise id="5" title="True or False: Visualization Libaries">
-
-
-**True or False**    
-*A low-level visualization library focuses on plot construction details.*
-
-<choice id="1" >
+<choice id="3" >
 <opt text="True"  correct="true">
 
-Great job! 
+You got it!
 
 </opt>
 
 <opt text="False">
 
-Hmmm. It's possible you got mixed up with high-level visualization. 
+They are! Volunteer contributions from people around the world support the development of these tools.
 
 </opt>
 
 </choice>
 
-**True or False**    
-*A high-level visualization library focuses on data and relationships.*
-
-<choice id="2" >
-
-<opt text="True"  correct="true">
-
-Nice work!
-
-</opt>
-
-<opt text="False">
-
-High-level visualization libraries concentrate on the data and the relationships between them. 
-
-</opt>
-
-</choice>
 </exercise>
 
-
-<exercise id="6" title="Your First Plot">
-
-**Instructions:**    
-Be patient when running a coding exercise for the first time, it can take a few minutes. 
-
-**When you see `____` in a coding exercise, replace it with what you assume to be the correct code. Run the code to see if you obtain the desired output
-and submit it to validate if you were correct.**
-
-
-Let's use the same vehicle dataset we saw in the lecture, but visualize the relationship between two different columns.
-
-The data has already been imported for you in this exercise.
-
-Here is what it looks like:
-
-<codeblock id="cars_data">
-
-</codeblock>
-
-Tasks: 
-
-- Fill out the missing fields in the Altair plot in order to create a scatter plot.
-- The cars' `Horsepower` should be on the y-axis and their `Weight_in_lbs` on the x-axis.
-- Color the points by the `Origin` of the cars.
-
-<codeblock id="01_06">
-
-- Are the column names expressed in quotes, e.g. `y='Horsepower'`?
-
-</codeblock>
+<exercise id="3" title="How Can We Visualize Data?" type="slides,video">
```

Comment:
This should be removed I guess?

Reply:
I was keeping this as a placeholder to have id=3 show up in the slides. But yes, the title should be changed based on what goes into slide section 3.